n2survey/plot/simple_comparison.py

```diff
@@ -0,0 +1,241 @@
+    return x_sorted, y_sorted
+
+
+def simple_comparison_plot(
+    df,
+    answer_suppress: list = None,
+    totalbar: np.ndarray = None,
+    bar_positions: Union[list, bool] = False,
+    threshold_percentage: float = 0,
+    bar_width: float = 0.8,
+    legend_columns: int = 2,
+    plot_title: Union[str, bool] = False,
+    answer_sequence: Union[list, bool] = False,
+):
+    """
+    plots correlations from the np.ndarray df with the existing answer
+    combinations and applies the given specifications.
+    """
+    if answer_suppress is None:
+        answer_suppress = []
+    # form x-axis with answers to first question and y-axis with
+    # percentages of second question correlated to each answer of the first
+    # question.
+    (x, y) = form_x_and_y(df, totalbar=totalbar, answer_suppress=answer_suppress)
+    # complete and filter answer sequence after sorting out answers
+    answer_sequence = filter_answer_sequence(x, answer_sequence)
+    # create a list with positonings of the bars in the plot.
+    bar_positions_complete = form_bar_positions(
+        df, bar_positions, totalbar=totalbar, answer_suppress=answer_suppress
+    )
+    # sort x and y to follow answer_sequence
+    (x, y) = sort_data(answer_sequence, x, y)
+
+    # %% Prepare/Define figure
+    fig, ax = plt.subplots()
+    # %% split up y to list of answers of question 2 and list of percentages
+    q2_answers = []
+    percentages = []
+    for entry in y:
+        q2_answers.append(entry[:, 0])
+        percentages.append(entry[:, 1].astype(np.float64))
+    all_answers = np.unique(np.concatenate(np.array(q2_answers)))
+    percentage_all = []
+    for (percentage, q2_answer) in zip(percentages, q2_answers):
+        count = 0
+        percentage_all_single = []
+        for answer in all_answers:
+            if answer in q2_answer:
+                percentage_all_single.append(percentage[count])
+                count = count + 1
+            else:
+                percentage_all_single.append(0)
+        percentage_all.append(percentage_all_single.copy())
+    # %% plot and format
+    bottom = 0
+    count = 0
+    for answer, percentage in zip(all_answers, np.transpose(np.array(percentage_all))):
+        ax.bar(
+            bar_positions_complete,
+            percentage,
+            bottom=bottom,
+            label=answer,
+            width=bar_width,
+        )
+        plt.xticks(bar_positions_complete, x)
+        bottom = bottom + percentage
+        labels = percentage.astype(str)
+        labels[np.where(labels.astype(np.float64) <= threshold_percentage)] = ""
+        ax.bar_label(ax.containers[count], labels, fmt="%s", label_type="center")
+        count = count + 1
+    labels = all_answers
+    ax.legend(
+        labels, bbox_to_anchor=([0.1, 1, 0, 0]), ncol=legend_columns, frameon=False
+    )
+    ax.axes.get_yaxis().set_visible(False)
+    if plot_title:
+        ax.set_title(plot_title, pad=60)
```

Comment:
Added the option `plot_title_position` also if it is not given, the title position is now calculated from number of entries, number columns of the entry and the existence of `legend_title`.

I think the Question to the y-axis of the plot should be the title of the Plot since it is the focus of the Plot, of course the additional questions are not considered automatically, in that case the user has to make his own title.
But I added the option `legend_title` (True, None, str) to add either the `compare_with=` question or one of own choosing, this option is also considered in the calculation of the padding title.